src/Client.js

```diff
@@ -326,8 +326,15 @@ Client.prototype._execute = function(method, path, data, query, options) {
         endTime
       )
       self._handleRequestResult(response, result, options)
-
-      return responseObject['resource']
+      
+      if (options.withFaunaMetrics) {
+        return { 
+          response: responseObject['resource'],
+          headers: response.headers
```

Comment:
headers !== metrics. Sure, metrics are part of the headers, but not all of the response headers are metrics.

We should return `metrics`, with the subset of the headers that represent metrics.

The headers that represent metrics are:

```
[
  'x-query-bytes-in',
  'x-query-bytes-out',
  'x-query-time',
  'x-read-ops',
  'x-write-ops',
  'x-compute-ops',
  'x-storage-bytes-read',
  'x-storage-bytes-write',
  'x-txn-retries',
]
```

Reply:
Is there a test that exercises the metrics return? If not, shouldn't there be at least one?

Reply:
The JVM client's [MetricsResponse class](https://github.com/fauna/faunadb-jvm/blob/5ffdba7c0ffe8e642c7c2ed2131d41d677de8037/faunadb-java/src/main/java/com/faunadb/client/types/MetricsResponse.java#L13-L15) has `metricsMap` and `value` fields.  The "Map" part is redundant in JS, so if we want to harmonize with JVM driver, then we could return 

```javascript
{
  value:  responseObject['resource'],
  metrics: getMetrics(response.headers)
}
```

Also highlighting that headers `!==` metrics

Reply:
Done